db/migrate/20150630221004_add_public_to_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddPublicToProfiles < ActiveRecord::Migration
+  def change
+    add_column :profiles, :is_public, :boolean, :default => false
```

Comment:
Can't we drop the `is_` prefix here and use the predicate accessor so it becomes idiomatic ruby, `.public?`?